doc/fwutil/fwutil.md

```diff
@@ -257,22 +280,55 @@ Chassis   Module   Component   Firmware               Version (current/available
 Chassis1  N/A      BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
                    CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
 ```
 
-2. Modular chassis platform
+2. Non modular chassis platform when the "boot" key and value are available in `platform_components.json`
+```bash
+root@sonic:~# fwutil show updates --image=next
+Chassis   Module   Component   Firmware               Version (current/available)              Status
+--------  -------  ----------  ---------------------  ---------------------------------------  ------------------
+Chassis1  N/A      BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
+                   CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
+```
+
+3. Modular chassis platform
 ```bash
 root@sonic:~# fwutil show updates --image=next
 Chassis   Module   Component   Firmware               Version (current/available)              Status
 --------  -------  ----------  ---------------------  ---------------------------------------  ------------------
 Chassis1           BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
                    CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
           Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
 ```
 
+4. Custom FW Package when the "boot" key and value are available in `platform_components.json`
```

Comment:
@sujinmkang What would be the CLI output if we don't have these available:
`when the "boot" key and value are available`

Reply:
Good catch. we are not going to have the boot key in platform_components.json. The platform auto-update api will perform the expected firmware update based on the boot type.  So we don't need to have the boot key in platform_components.json.